plugin/src/main/groovy/grails/plugin/springsecurity/ui/SpringSecurityUiGrailsPlugin.groovy

```diff
@@ -34,7 +34,7 @@ import groovy.util.logging.Slf4j
 @Slf4j
 class SpringSecurityUiGrailsPlugin extends Plugin {
 
-	String grailsVersion = '3.0.0 > *'
+	String grailsVersion = '7.0.0 > *'
```

Comment:
I think this has to be '7.0.0-SNAPSHOT > *' to work with snapshots.

Reply:
That is correct. As is will only work with final 7 release  https://github.com/codeconsole/grails-plugin-sitemesh3/blob/b2f55a1f96c24a58a0e33dd7e331e32fe0d0ca5d/src/main/groovy/org/sitemesh/grails/plugins/sitemesh3/Sitemesh3GrailsPlugin.groovy#L13

Reply:
Is this enforced somewhere?